lib/app_worker.js

```
@@ -35,6 +35,7 @@ const clusterConfig = app.config.cluster || /* istanbul ignore next */ {};
 const listenConfig = clusterConfig.listen || /* istanbul ignore next */ {};
 const httpsOptions = Object.assign({}, clusterConfig.https, options.https);
 const port = options.port = options.port || listenConfig.port;
+const reusePort = options.reusePort = options.reusePort || listenConfig.reusePort;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)